### PR TITLE
sparse: update to clangarm64 paths

### PR DESCRIPTION
**Pull request by dennisameling**
This prepares the `git-sdk-arm64` for being able to build SDK artifacts.

I built the artifacts locally through e.g. `/usr/src/build-extra/please.sh create-sdk-artifact --architecture=aarch64 --out=/sdk-build-installers build-installers`:

- `build-installers` ✅ (784MB)
- `makepkg-git` ✅ (602MB)
- `minimal` ✅ (415MB)

I used `build-installers` to build Git as follows: `/usr/src/build-extra/please.sh build-mingw-w64-git --only-aarch64 -o artifacts HEAD`. That completed successfully 🎉

### CODE REVIEW

File: .sparse/makepkg-git [hunk 1]
```diff
@@ -52,33 +52,16 @@
...
+# AsciiDoc
+/clangarm64/bin/asciidoc.exe
+/clangarm64/bin/asciidoc-script.py
+/clangarm64/bin/python*
+/clangarm64/bin/libpython*.dll
+/clangarm64/lib/python*/
+!/clangarm64/lib/python*/site-packages/
+!/clangarm64/lib/python*/test/
+/clangarm64/lib/python*/site-packages/asciidoc/
+/clangarm64/lib/python*/site-packages/asciidoc*-info/
```
Comment (dennisameling) [inline .sparse/makepkg-git:64]: I tried to be more mindful here by including only specific folders from `/clangarm64/lib/python`, but the amount of folders became so large that I just chose to include the entire folder instead and exclude the largest unnecessary ones. With this config, the entire `/clangarm64/lib/python3.10` folder is ~26MB, so I think that's ok.

File: .sparse/minimal-sdk [hunk 2]
```diff
@@ -2,134 +2,97 @@
...
+/clangarm64/bin/libgmp-*[0-9].dll
+/clangarm64/bin/libclang.dll
+/clangarm64/bin/libclang-cpp.dll
+/clangarm64/bin/libLLVM-*[0-9].dll
+/clangarm64/bin/libwinpthread-*[0-9].dll
+/clangarm64/bin/libzstd.dll
+/clangarm64/bin/llvm-strip.exe
+/clangarm64/bin/windres.exe
+/clangarm64/lib/clang/*/include/
+/clangarm64/lib/clang/*/lib/
```
Comment (dennisameling) [inline .sparse/minimal-sdk:30]: Is all of this really needed in the minimal SDK? `libLLVM-15.dll` alone is ~96MB already...

Comment (dscho) [reply .sparse/minimal-sdk:30]: I fear that's the price of using a low-level virtual machine ;-)

File: .sparse/makepkg-git (hunk 1 above)
Comment (dscho) [reply .sparse/makepkg-git:64]: Sure. An alternative might be to use `asciidoctor` instead, which uses Ruby, like we use to build `mingw-w64-x86_64-git-doc-html`...

**Review by dscho — APPROVED** [review]

File: .sparse/makepkg-git (hunk 1 above)
Comment (dennisameling) [reply .sparse/makepkg-git:64]: I'd love to, but `asciidoctor` [isn't available yet](https://github.com/git-for-windows/MINGW-packages/blob/147af4445ca6af015b88dd87a466591d88324e2a/mingw-w64-git/PKGBUILD#L101) for `clang-aarch64` 😅

**Pull request merged**